pabi_hr_expense/models/hr_expense.py

```diff
@@ -153,6 +153,30 @@ class HRExpense(models.Model):
     )
 
     @api.multi
+    @api.constrains('line_ids')
+    def _check_line_ids(self):
+        chart_fields = {
+            'project_id': [],
```

Comment:
You can use following code to get all project_id.
{
    'project_id': self.advance_expense_id.line_ids.mapped('project_id')._ids,